Zend/zend_object_handlers.c

```diff
@@ -537,8 +537,11 @@ zval *zend_std_read_property(zval *object, zval *member, int type, void **cache_
 	/* make zend_get_property_info silent if we have getter - we may want to use it */
 	property_offset = zend_get_property_offset(zobj->ce, Z_STR_P(member), (type == BP_VAR_IS) || (zobj->ce->__get != NULL), cache_slot);
 
+
 	if (EXPECTED(property_offset != ZEND_WRONG_PROPERTY_OFFSET)) {
 		if (EXPECTED(property_offset != ZEND_DYNAMIC_PROPERTY_OFFSET)) {
+            EG(current_execute_data)->func->op_array.accessor.property_offset = property_offset;
```

Comment:
It may be not safe to use op_array fields as cache. It's possible to use a slot from op_array->run_time_cache[].